cmd/process-agent/collector_api_test.go

```diff
@@ -0,0 +1,198 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util/api"
+
+	"github.com/DataDog/agent-payload/process"
+
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSendMessage(t *testing.T) {
```

Comment:
nit: Shouldn't this be called `TestSendConnectionMessage`?

Reply:
It can be, yes.  The goal for this was just a sanity check test and I chose a connection method for that purpose but it could really be any of the types.